assets/styles/authoring/flowchart.scss

```diff
@@ -675,16 +673,16 @@ $hover: #dce7f9;
   .aa-stage {
     position: fixed;
     z-index: 9;
-    top: 210px;
-    left: 335px;
+    top: 156px;
+    left: 180px;
     right: 270px;
     bottom: 0;
     overflow: scroll;
     background-color: #f3f5f8;
 
     .aa-stage-pan {
-      width: 2000px;
-      height: 1000px;
+      width: auto;
```

Comment:
Setting the height and width to auto because it was causing a vertical and horizontal scroll bar even though there were very less parts on the screen